server/plugin/plugin.go

```diff
@@ -250,6 +263,86 @@ func (p *Plugin) OnDeactivate() error {
 	return nil
 }
 
+func (p *Plugin) ReactionHasBeenAdded(c *plugin.Context, reaction *model.Reaction) {
+	githubEmoji := p.emojiMap[reaction.EmojiName]
+	if githubEmoji == "" {
+		return
+	}
+	post, error := p.client.Post.GetPost(reaction.PostId)
+	if error != nil {
+		p.API.LogError("Invalid post id. Addition of reaction failed.")
+		return
+	}
+
+	repo := post.GetProp("gh_repo")
+	if repo == nil {
+		p.API.LogDebug("Required prop is not present. This post is not created by this plugin, hence ignoring")
+		return
+	}
+
+	orgRepo := strings.Split(repo.(string), "/")
```

Comment:
I suggest we make the code more compact by doing nil checking and type assertion in the same line. The `ok` syntax is the "safer" way of doing a type assertion. Otherwise if `repo` is not a string, the program will panic. That is highly unlikely here but should always be treated as possible.

```suggestion
	repo, ok := post.GetProp("gh_repo").(string)
	if !ok || repo == "" {
		return
	}
	
	orgRepo := strings.Split(repo, "/")
```

Reply:
```suggestion
	orgRepo := strings.Split(repo.(string), "/")
	if len(orgRepo) != 2 {
	    // handle error
	}
```